### ISSUE TITLE
Vim(let):E716: Key not present in Dictionary: "headers.subject" 

### ISSUE THREAD

**Issue by TamasSmahajcsikszabo** [soywod/himalaya#208]
Hi, 
I'm using neovim 0.5.0 and recently installed himalaya 0.4.0. 
CLI works fine and prints inbox of my Outlook account, but in nvim, if I call `:Himalaya`, I get:
`Vim(let):E716: Key not present in Dictionary: "headers.subject"` 

Could you please advise?

Thank you,
Tamas

**Comment by TamasSmahajcsikszabo** [comment]
With Vimplug, such as `Plug 'soywod/himalaya', {'rtp': 'vim'}   `

**Comment by soywod** [comment]
The problem is that the CLI is versioned but not the Vim plugin (since it clones the git repo). For now, I would advise to align the CLI version with the Vim plugin branch. If you use the `v0.4.0` of the CLI then use the `v0.4.0` branch/tag from the Vim plugin instead. What I could do is to use the `master` branch as the production one (the last master commit = a tagged version) and develop on a `development` branch. If you have better idea, let me know!

**Comment by wjcagle** [comment]
I just tried making sure they're both aligned. I've got the v0.4.0 versions for both. I'm still getting the same error.

**Comment by soywod** [comment]
Do you also use `vim-plug`? If so then could you add the `tag` entry in your plug command? Sth like `Plug 'soywod/himalaya', {'rtp': 'vim', 'tag': 'v0.4.0'}` and try again?

**Comment by wjcagle** [comment]
I do. Let me give that a try.

**Comment by wjcagle** [comment]
Looks like it's working now. I had to totally delete my HImalaya plugin and let it re-download complately. but it's working now. Thank you!

**Comment by wjcagle** [comment]
OK, it seems I'm having other issues now. If I try to write a message, for example, I get Vim(call):E716: Key not present in Dictionary: "template, "\r", "", "g"), "\n"))"

**Comment by TamasSmahajcsikszabo** [comment]
@soywod I did what you've advised above (aligning the plugin version to the CLI; both at 0.4.0) and it solved the issue on my end. Himalaya is working as expected. Thank you for the help!

**Comment by wjcagle** [comment]
@TamasSmahajcsikszabo No problems writing e-mail or anything? Because I'm having problems with that now.

**Comment by TamasSmahajcsikszabo** [comment]
@wjcagle Inbox search, listing work well, but when writing new messages, I can confirm your error message. I get the same error, such as: 
`Vim(call):E716: Key not present in Dictionary: "template, "\r", "", "g"), "\n"))" `

**Comment by soywod** [comment]
Then I advise you to wait for the `v0.5.0`, I should release it in the next days. Sorry for those bugs, the project is still young!

**Comment by TamasSmahajcsikszabo** [comment]
@soywod No worries, thank you, keep up the good work.

**Comment by soywod** [comment]
Thanks for the kind words. I let you know when the new release is available!

**Comment by wjcagle** [comment]
Agreed. Keep up the good work, and I'll look forward to v.0.5.0. :)
